test/helpers/fixture_flows_helper.rb

```diff
@@ -21,4 +33,15 @@ def stub_smart_answer_in_content_store(smart_answer_id)
       .with("/#{smart_answer_id}")
       .returns({})
   end
+
+  def button_text_and_link(smart_answer_slug)
```

Comment:
Same as the comment above, why do we have this method here? It is duplicating the behaviour specified in the [application helper][1]. 

[1]: https://github.com/alphagov/smart-answers/blob/432830cf9b3e9dbafbe4c63d886b29b900947151/app/helpers/application_helper.rb#L14-L31